package.json

```diff
@@ -27,13 +27,15 @@
     }
   },
   "dependencies": {
-    "@gnosis.pm/dex-contracts": "^0.2.10",
+    "@gnosis.pm/dex-contracts": "file:/home/vk/Gnosis/dex-contracts",
+    "@types/workerpool": "^5.0.1",
     "@types/yargs": "^15.0.4",
```

Comment:
nit: These should be `devDependencies` since they are only used for compiling.

Reply:
And it only needs the types during compilation but the real workerpool when running. I see.

Reply:
:+1: The `@types/*` packages typically only contain `.d.ts` files which are type definitions for describing JS libraries, but don't actually contain any code or a module that gets included at runtime.